docs/protocol/substrate-cli.mdx

```diff
@@ -924,6 +928,37 @@ Example:
     path=/media/ssd3,size=10T
 ```
 
+### Optimizing DSN Syncing
+
+:::warning Parameters to Use with Caution
+While it might be tempting to adjust certain parameters that seem related to DSN Syncing, it's advised to be cautious. Some options might not enhance syncing and are best left at their default values:
+
+```
+--out-peers
+--in-peers
+--in-peers-light
+--dsn-target-connections
+--dsn-pending-in-connections
+--dsn-in-connections
+```
+:::
+
+#### Recommended Parameters
+
+The default parameters are set with the capabilities of common consumer modem/routers in mind. Adjusting certain parameters can enhance DSN sync performance. However, if you decide to increase these parameters significantly, ensure that your modem/router is performant enough to handle the increased traffic.
+
+**Node:**
+```
+--dsn-out-connections
+--dsn-pending-out-connections
+```
+
+**Farmer:**
+```
+--out-connections
+--pending-out-connections
+```
+
```

Comment:
Can we add "Increasing the values of the farmer parameters could increase the plotting speed." please?